server/datastore/mysql/activities_test.go

```diff
@@ -55,6 +59,32 @@ func (d dummyActivity) Documentation() (activity string, details string, details
 	return "", "", ""
 }
 
+type fakeHostActivity struct {
```

Comment:
no action required, I wonder if we should add `hostIDs` to `dummyActivity` above instead.